src/lib/migration-chunks/validation/tag.ts

```diff
@@ -32,7 +32,66 @@ class DuplicateCreate implements TagValidation {
   }
 }
 
-class AlreadyExistingCreates implements TagValidation {
+class EditBeforeCreates implements TagValidation {
```

Comment:
Feels like all of these have a pretty close match to what content types do.
Let's keep them separate for now but keep in mind the opportunity for extracting this behavior for the next entity that needs this kind of logic.

Reply:
Yes, I agree. Ideally we would have an EntityValidation which could be made used of by Tags and ContentTypes and whatever else is coming.